apps/native/app/src/screens/air-discount/air-discount.tsx

```diff
@@ -0,0 +1,278 @@
+import {
+  Alert,
+  EmptyListSmall,
+  Heading,
+  Link,
+  LinkText,
+  Skeleton,
+  Typography,
+} from '@ui'
+import React from 'react'
+import { SafeAreaView, ScrollView, View, Image } from 'react-native'
+import { NavigationFunctionComponent } from 'react-native-navigation'
+import { FormattedMessage, useIntl } from 'react-intl'
+import styled, { useTheme } from 'styled-components/native'
+
+import { createNavigationOptionHooks } from '../../hooks/create-navigation-option-hooks'
+import {
+  useGetAirDiscountFlightLegsQuery,
+  useGetAirDiscountQuery,
+} from '../../graphql/types/schema'
+import { AirDiscountCard } from '@ui/lib/card/air-discount-card'
+import { Bullet } from '@ui/lib/bullet/bullet'
+import { AirfaresUsageTable } from './airfares-usage-table'
+import illustrationSrc from '../../assets/illustrations/le_jobs_s5.png'
+
+const BulletList = styled.View`
+  margin-vertical: 12px;
+`
+
+const TOSLink = styled.View`
+  margin-top: ${({ theme }) => theme.spacing[1]}px;
+  margin-bottom: ${({ theme }) => theme.spacing[2]}px;
+`
+
+const AlertWrapper = styled.View`
+  margin-top: ${({ theme }) => theme.spacing[1]}px;
+  margin-bottom: ${({ theme }) => theme.spacing[3]}px;
+`
+const { getNavigationOptions, useNavigationOptions } =
+  createNavigationOptionHooks((theme, intl) => ({
+    topBar: {
+      title: {
+        text: intl.formatMessage({ id: 'airDiscount.screenTitle' }),
+      },
+    },
+  }))
+
+const SkeletonItem = () => {
+  const theme = useTheme()
+  return (
+    <View>
```

Comment:
Is the View necessary?
I have not seen the convention in the project to have multiple components in one file. The SkeletonItem is only used once in this component so you could also just have it in the main component.

Reply:
I was following the pattern I found [here](https://github.com/island-is/island.is/blob/main/apps/native/app/src/screens/vehicles/vehicles.tsx#L61)

Reply:
And no the view is not necessary, I had some style on it that I then removed - good catch :)